dmutils/forms/helpers.py

```diff
@@ -17,3 +19,21 @@ def remove_csrf_token(data):
         del cleaned_data['csrf_token']
 
     return cleaned_data
+
+
+def govuk_option(option: typing.Dict) -> typing.Dict:
```

Comment:
If we're not being more specific about the type of Dict we can just write `dict` and save ourselves an import.

```suggestion
def govuk_option(option: dict) -> dict:
```